packages/components/src/toggle-group-control/toggle-group-control-option-base/component.tsx

```diff
@@ -123,6 +146,19 @@ function ToggleGroupControlOptionBase(
 					</Radio>
 				) }
 			</WithToolTip>
+			{ /* Animated backdrop using framer motion's shared layout animation */ }
+			{ isPressed ? (
+				<motion.div
```

Comment:
Do we need to tweak the transition duration to match the original (`transitionDurationFast`)? It feels a lot slower than what we have in trunk.

Reply:
Joen seems ok with this in [his review above](https://github.com/WordPress/gutenberg/pull/50278#pullrequestreview-1471390529)

> The animation is also slightly slower, that's fine.